plugin-templates/entries/slotfills/featured-image-caption/FeaturedImageCaption.tsx

```diff
@@ -0,0 +1,21 @@
+import { TextareaControl } from '@wordpress/components';
```

Comment:
Curious as to how this can support cases where the plugin doesn't have a dependency used in a feature? Could we run a script to install a missing dependency?

Reply:
I created https://github.com/alleyinteractive/create-wordpress-project/issues/59 to handle this.